src/muc_light/mod_muc_light.erl

```diff
@@ -378,66 +376,78 @@ make_err(From, To, El, Acc, Reason) ->
 %% Hook handlers
 %%====================================================================
 
--spec prevent_service_unavailable(Acc :: map(), From ::jid:jid(), To ::jid:jid(),
-                                  Packet :: exml:element()) -> map() | {stop, map()}.
-prevent_service_unavailable(Acc, _From, _To, Packet) ->
+-spec prevent_service_unavailable(Acc, Params, Extra) -> {ok | stop, Acc} when
+    Acc :: mongoose_acc:t(),
+    Params :: map(),
+    Extra :: map().
+prevent_service_unavailable(Acc, #{packet := Packet}, _Extra) ->
     case xml:get_tag_attr_s(<<"type">>, Packet) of
         <<"groupchat">> -> {stop, Acc};
-        _Type -> Acc
+        _Type -> {ok, Acc}
     end.
 
--spec disco_local_items(mongoose_disco:item_acc()) -> mongoose_disco:item_acc().
+-spec disco_local_items(Acc, Params, Extra) -> {ok | stop, Acc} when
```

Comment:
This handler will actually never return `{stop, Acc}`. Maybe we should change the return type of this spec to just `{ok, Acc}`? If you agree then please change it for other handlers which don't use `stop` atom.